common/views/components/UserProvider/UserProvider.tsx

```diff
@@ -32,7 +32,8 @@ const UserProvider: FC = ({ children }) => {
   const [state, setState] = useState<State>('initial');
 
   const fetchUser = async (abortSignal?: AbortSignal, refetch = false) => {
-    setState('loading');
+    setState('failed');
```

Comment:
this was to shut up the identity stuff in the console, not for any bigger reason